js/jquery.colInherit.js

```diff
@@ -32,8 +32,8 @@
                     css[selector] = {};
 
                     style.split(';').forEach(function(declaration) {
```

Comment:
@bvw @PeterChaplin 
```
var property = declaration.split(':')[0].trim();
var value = declaration.split(':')[1].trim();
```

This breaks when style is `width:70px;"

due to `style.split(";")` declaration[1] = ""
then `"".split(':')[1]` is undefined and causes error when doing trim as trim only works on strings.

```
Note: This API has been deprecated in jQuery 3.5; please use the native String.prototype.trim method instead. Unlike jQuery.trim, String.prototype.trim does not work with types other than strings (null, undefined, Number). Make sure that your code is compatible when migrating.
```

Please correct and check other cases `.trim()` is used in this pull request to ensure `.trim()` is only used on strings.